modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLValidatorImpl.java

```diff
@@ -93,6 +94,25 @@ public long getMaxAllowableSize(long groupId, String mimeType) {
 					groupId, mimeType)));
 	}
 
+	@Override
+	public Map<String, Long> getMimeTypeSizeLimit(long groupId) {
+		long companyId = _getCompanyId(groupId);
+
+		Map<String, Long> mimeTypeSizeLimit = new HashMap<>(
+			_dlSizeLimitManagedServiceFactory.getGroupMimeTypeSizeLimit(
+				groupId));
+
+		Map<String, Long> companyMimeTypeSizeLimit = new HashMap<>(
```

Comment:
In my opinion, if the companyMimeTypeSizeLimit is not modified, there is no need to create a new Map with the containing of the _dlSizeLimitManagedServiceFactory.getCompanyMimeTypeSizeLimit

Reply:
agree with you, I updated it